Makefile

```diff
@@ -65,7 +65,46 @@ python_sdk::
 		rm ./bin/setup.py.bak && \
 		cd ./bin && python3 setup.py build sdist
 
+gen_examples::
+	cd ${WORKING_DIR}/examples/yaml && \
+	rm -rf ${WORKING_DIR}/examples/{go,nodejs,python,dotnet} && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language go --out ${WORKING_DIR}/examples/go 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language nodejs --out ${WORKING_DIR}/examples/nodejs 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language python --out ${WORKING_DIR}/examples/python 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language dotnet --out ${WORKING_DIR}/examples/dotnet 2>&1
+
+pulumi_login::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	pulumi login --local
+
+pulumi_up::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack init dev && \
+	pulumi stack select dev && \
+	pulumi config set name dev && \
+	pulumi up -y
+
+pulumi_down::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack select dev && \
+	pulumi destroy -y && \
+	pulumi stack rm dev -y
```

Comment:
I don't understand what the purpose of this is. What part of the development cycle should this be used in?

Reply:
I updated these targets in commit https://github.com/pulumi/pulumi-provider-boilerplate/pull/80/commits/cf48af22aa0e5913d3a5fa54c2b30d309df7abc9.

These targets are useful when developing the base examples/yaml test IaC example using the locally built provider from which the remaining languages example IaC code is generated.

I wrote them in when Runpod asked how to test their provider locally as they build it.